examples/tracing/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+services:
+  # Jaeger
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - "16686:16686"
+      - "14268"
+      - "14250"
+
+  #Zipkin
+  zipkin:
+    image: openzipkin/zipkin
+    container_name: zipkin
+    ports:
+      - 9411:9411
```

Comment:
Do we need zipkin here considering jaeger all in one can accept zipkin and show the traces as well?

Reply:
It's not required but it's neat to see OTel collector serving its vision to drive multiple backends, and it's helpful to be able to compare the views to make sure the exported data is sane. The non-contrib demo similarly starts them both https://github.com/open-telemetry/opentelemetry-collector/blob/master/examples/demo/docker-compose.yaml#L12